src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -20937,10 +20937,10 @@ public C()
                 );
         }
 
-        [Fact]
-        public void MemberNotNull_LocalFunction()
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/56256")]
+        public void MemberNotNull_InstanceLocalFunction()
```

Comment:
>InstanceLocalFunction [](http://example.com/codeflow?start=34&length=21)

Consider naming this `_NonStaticLocalFunction`, for consistency with other tests, and because there's no requirement that the local function takes an instance (and doesn't in this case). There are a couple of other uses of `_InstanceLocalFunction` below as well.

Reply:
Yup. Sorry, I missed those